<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolverSettingsDialog</class>
 <widget class="QDialog" name="SolverSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>199</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>255</width>
    <height>199</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxAuto">
     <property name="text">
      <string>Automatic mode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Altitude</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QDoubleSpinBox" name="AltitudeSpinBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Track length</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QDoubleSpinBox" name="TrackLenSpinBox"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Samples count</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDoubleSpinBox" name="SamplesCountSpinBox"/>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="AcceptButton">
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="CancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="ClearButton">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
